### PR TITLE
Added Dynamic Picture frame

### PR DESCRIPTION
**Pull request by Whizz76**
### 🛠️ Fixes Issue (618)

fixed issue #618 

### 👨‍💻 Changes proposed

Added index.html file
Added style.css file
Added script.js file

### ✅ Check List (Check all the applicable boxes) 

- ✅My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- ✅This PR does not contain plagiarized content.
- ✅ My Addition/Changes works properly and matches the overall repo pattern.
- ✅ The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

Added the dynamic Picture Frame

### 📷 Screenshots
pic

### CODE REVIEW

File: cards.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
         "image": "assets/Images/accordion.png",
         "link": "Public/accordion.html"
     },
+ {
```
Comment (Astrodevil) [inline cards.json]: it's not properly added, also follow alphabetical order.

**Comment by Astrodevil** [comment]
Why are you creating single folder for all files, please put your css, html, images in repo directory properly.

**Comment by Whizz76** [comment]
Had made the changes, please review it

File: cards.json (hunk 1 above)
Comment (Whizz76) [reply cards.json]: Have made the changes. Please review it.

**Comment by Whizz76** [comment]
changed ref link names in index.html

**Comment by Whizz76** [comment]
further changed

**Comment by Astrodevil** [comment]
You have done so many commits in 1 PR. please update your fork. Some errors are coming by my side. @Whizz76

**Comment by Astrodevil** [comment]
Please make a fresh PR with proper changes. Don't made multiple commits (Try to squash into one)

**Pull request closed without merging**